channeldb/revocation_log.go

```diff
@@ -204,6 +204,26 @@ type RevocationLog struct {
 	// HTLCEntries is the set of HTLCEntry's that are pending at this
 	// particular commitment height.
 	HTLCEntries []*HTLCEntry
+
+	// LocalBalance is the current available balance within the channel
```

Comment:
what happens to channels that are currently open? so they will have mixed revocation logs?

Reply:
yes they will unfortunately. Callers should be aware of this and always know how to handle the `ErrRevLogDataMissing` error